PROVISIONING-METAL.md

```diff
@@ -0,0 +1,169 @@
+# Povisioning Bottlerocket on metal
+
+This guide will describe what is needed to properly provision Bottlerocket on bare metal.
+Provisioning Bottlerocket on metal is different than provisioning other general-purpose distros.
+Since Bottlerocket has a `dm-verity`-checked boot and root partition, and is immutable at runtime, we cannot simply provision an image and directly write configuration files.
+Bottlerocket requires a few files to be generated and written to disk at provisioning time in order to boot properly; these files are described below.
+
+> For more information about the hardware that Bottlerocket is currently tested on, see [Hardware](HARDWARE.md).
+
+## Prerequisites
+
+* Bottlerocket metal image
+  * [`tuftool`](https://github.com/awslabs/tough/blob/develop/tuftool/README.md) is required to fetch the image
+* [User data TOML file](#user-data): `user-data.toml`
+* [Network configuration TOML file](#network-interface-configuration): `net.toml`
+* (Optional) [Boot configuration file](#boot-configuration): `bootconfig.data`
+
+### Fetch the Bottlerocket metal image
+
+The Bottlerocket metal image is signed and uploaded alongside the rest of the Bottlerocket release artifacts.
+
+You first need the Bottlerocket root role, which is used by `tuftool` to verify the image.
+The following will download and verify the root role itself:
+
+```
+curl -O "https://cache.bottlerocket.aws/root.json"
+sha512sum -c <<<"e9b1ea5f9b4f95c9b55edada4238bf00b12845aa98bdd2d3edb63ff82a03ada19444546337ec6d6806cbf329027cf49f7fde31f54d551c5e02acbed7efe75785  root.json"
+```
+
+Next, set your desired version and variant, and download the image:
+```
+VERSION="1.8.0"
+VARIANT="metal-k8s-1.23"
+SHA="a6233c22"
+IMAGE="bottlerocket-${VARIANT}-x86_64-${VERSION}-${SHA}.img.lz4"
+OUTDIR="${VARIANT}-${VERSION}"
+
+tuftool download "${OUTDIR}" --target-name "${IMAGE}" \
+   --root ./root.json \
+   --metadata-url "https://updates.bottlerocket.aws/2020-07-07/${VARIANT}/x86_64/" \
+   --targets-url "https://updates.bottlerocket.aws/targets/"
+```
+
+### User data
+
+For metal, Bottlerocket expects a TOML-formatted file named `user-data.toml` that contains user data settings.
+Acceptable settings can be found in the [settings docs](https://github.com/bottlerocket-os/bottlerocket#settings).
+
+If you're just getting started and want to provision a host without connecting to a kubernetes cluster, you can use the following example user data which will start kubelet in standalone mode.
+
+```toml
+[settings.kubernetes]
+standalone-mode = true
+```
+
+It's important to note for remote access to your running Bottlerocket hosts, you will need to add user data to enable host containers.
+The Bottlerocket metal images don't enable any host containers by default.
```

Comment:
As a general editorial style note, I'd recommend coming up with 2-3 phrases and using them consistently:
* "Bottlerocket images for bare metal"
* "Bottlerocket for bare metal"

Currently it reads kind of sloppy and imprecise because we switch between talking about "Bottlerocket" and various permutations of "Bottlerocket for bare metal", and it's not clear when the reader is learning something that's generally applicable to all of Bottlerocket, versus something that's specific to bare metal.